.github/workflows/build-docs.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - master
```

Comment:
should we remove this then?  there's no staging env, right?

Reply:
you'll need a section to actually call into vercel when the push happens to master then, without the `--prod` args.  You can map to an alias with the `master` subdomain.

Reply:
ohh i see - cos in the later commands we only handle pull_request and the docs-prod branch right?

Reply:
yeah, exactly.

Reply:
updated. expanding the preview build commands to pushes on master branch should do it right?